cmd/githubwebhookserver/main.go

```diff
@@ -124,7 +124,7 @@ func main() {
 	if watchNamespace == "" {
 		logger.Info("-watch-namespace is empty. HorizontalRunnerAutoscalers in all the namespaces are watched, cached, and considered as scale targets.")
 	} else {
-		logger.Info("-watch-namespace is %q. Only HorizontalRunnerAutoscalers in %q are watched, cached, and considered as scale targets.")
+		logger.Info("-watch-namespace is %q. Only HorizontalRunnerAutoscalers in %q are watched, cached, and considered as scale targets.", watchNamespace)
```

Comment:
```suggestion
		logger.Info("-watch-namespace is %q. Only HorizontalRunnerAutoscalers in %q are watched, cached, and considered as scale targets.", watchNamespace, watchNamespace)
```

Apparently we need one more!

Reply:
@PiotrPalkaSpotOn Hey! Unfortunately, I couldn't push an additional commit to your branch perhaps due to your org/repo settings. Could you address my above comment so that we can approve/merge this? Thanks.

Reply:
@mumoshu all done, thanks for finding a flaw :)